shibata revised for #37, 38, 39,40 (#42)
fujitake/vantiq-related: vantiq-apps-development/1-day-workshop/docs/eng/3-Lab02_Types.md

3. Open the "Properties" tab of the `Pumps` Type and use the _+ Add Property_ button to create the following four properties.  

| Name         | Data type | Note      |
|--------------|----------|-------------------|
| PumpID       | Integer  | ID of the Pump|
| Location     | GeoJSON  | Lon/Lat coordinates |
| TempSensorID | String   | ID of the Temperature sensor|

> | RPMSSensorID | String   | ID of the RPM sensor|

Reviewer: ```suggestion
| RPMSSensorID | String   | ID of the RPMS sensor|
```

> ＊ Saving a Type and saving a Project are different tasks. Right now, two Types have been created in the Namespace, but their relationship to the Project is not saved until the Project is saved.  

Reviewer: ```suggestion
＊ Saving a Type and saving a Project are different tasks. At this point, two Types have been created in the Namespace, but their relationship to the Project is not saved until the Project is saved.  
```

<img src="../../imgs/Lab02/image3.png" width=30%>

> ## ***Step 3 （Register data）***

Reviewer: ```suggestion
## ***Step 3 （Import master data）***
```

> 1. **Register data in Pumps Type**  

Reviewer: ```suggestion
1. **Import data in Pumps Type**  
```

Register master data in Pumps Type. This case, register the data with the procedure used when uploading multiple items of data at once.    
    1. Select "Projects" > "Import...", and open the "Import Projects" window. Change the "Select Import Type" from "Project" to "Data".  

>     2. Save the contents of the pre-distributed "[`Pumps.json`](https://github.com/fujitake/vantiq-related/raw/main/vantiq-apps-development/1-day-workshop/conf/Pumps.json)", create the JSON file, and drag and drop it into there.      

Reviewer: ```suggestion
    2. Save  "[`Pumps.json`](https://github.com/fujitake/vantiq-related/raw/main/vantiq-apps-development/1-day-workshop/conf/Pumps.json)" as a JSON file, and drag and drop it into there.      
```

    3. Click the _Import_ button.

2.  **Confirm the registered data**  

> Confirm that the data is correctly registered in the `Pumps` Type. Open `Pumps` Type, click the "Show All Records" and check if the data is registered like the following image.  

Reviewer: ```suggestion
Confirm that the data is correctly imported in the `Pumps` Type. Open `Pumps` Type, click the "Show All Records" and check if the data is imported as shown below.  
```
